angular/component-table/component-table-view.html

```diff
@@ -9,14 +9,14 @@
     <col width="100px" />
     <thead>
     <tr>
-       <th>  
+       <th>
         <th class="link" ng-click="setOrderProp('name')" ng-class="[{active: orderProp == 'name'},'listview-pointer']" > <nobr><i class="fa fa-bookmark-o"></i> Component</nobr>
         <!-- <th>Summary -->
 <!--         <th ng-click="setOrderProp('version')" ng-class="[{active: orderProp == 'version'},'listview-pointer']">Version
- -->    
+ -->
         <th class="link" ng-click="setOrderProp('stars')" ng-class="[{active: orderProp == 'stars'},'listview-pointer']"><nobr><i class="fa fa-star"></i> Stars</nobr>
-        <th class="link" ng-click="setOrderProp('downloads')" ng-class="[{active: orderProp == 'downloads'},'listview-pointer']"><nobr><i class="fa fa-download"></i> Downloads</nobr> 
-       
+        <th class="link" ng-click="setOrderProp('downloads')" ng-class="[{active: orderProp == 'downloads'},'listview-pointer']"><nobr><i class="fa fa-download"></i> Downloads</nobr>
+
```

Comment:
Oh, so it is a styling issue, sorry, I did not know about it. Indeed, better to get rid of those then.